packages/airnode-protocol/contracts/authorizers/mock/MockRequesterAuthorizerWithErc721.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "../../dev/RequesterAuthorizerWithErc721.sol";
+
+contract MockRequesterAuthorizerWithErc721 is RequesterAuthorizerWithErc721 {
```

Comment:
The `MockRequesterAuthorizerWithErc721` is not needed. You may just use `RequesterAuthorizerWithErc721` directly. I called it mock since it was only used for testing, but you may as well just use the real thing.

Reply:
Agreed- deleted.